src/app/frontend/deploy/deploy.scss

```diff
@@ -18,3 +18,10 @@
   // TODO(bryk): Find a better, application wide constant for the min width.
   min-width: 600px;
 }
+
+.kd-deploy-labels-button {
```

Comment:
Nice! Can you move this to deploylabel.scss? With this all deploylabel-related things are in deployalbel.* files. Then, e.g., when refactoring you know where to look for things.